Cows-HacknSlash/Assets/Scripts/Inventory script/Additem.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Additem : MonoBehaviour {
+
+    public Inventory invenscript;
+
+    public int ObjectID;
+
+    public int Objectamount = 1;
+	void Start () {
+        invenscript = GameObject.Find("PlayerCharacter/char").GetComponent<Inventory>();
+	}
+
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.gameObject.tag == "Player")
```

Comment:
Since tag is a string, please use `.equals("Player")` or `.ToString().equals("Player")` instead here.